src/main/java/com/doeiqts/pizzavoter/domain/Pizza.java

```diff
@@ -33,7 +34,7 @@ public Pizza(Size size, Crust crust, Sauce sauce) {
         toppingCount = 1.0;
     }
 
-    public Position addTopping(Topping topping, Position position) {
+    public Position addTopping(Topping topping, Position position, HttpServletRequest request) {
```

Comment:
Ugh, mixing front end and object stuff. Gross. Assuming this is functional, I'll probably change it later.